common-ktx/src/main/java/co/nimblehq/common/extensions/ViewExt.kt

```diff
@@ -49,14 +49,16 @@ fun View.visibleOrGone(visible: Boolean, delay: Long) {
     if (delay <= 0L) {
         visibleOrGone(visible)
     } else {
-        postDelayed({
-            val context: WeakReference<Context?>? = WeakReference(this.context)
-            if (context?.get() is Activity &&
-                (context?.get() as Activity).isFinishing) {
-                return@postDelayed
-            }
-            visibleOrGone(visible)
-        }, delay)
+        postDelayed(
+            {
```

Comment:
Why do we need to do a new line for this `{`? 🤔

Reply:
@lydiasama Yup, `detekt` shows a warning on this, that's why I have to break to a new line.

Reply:
@lydiasama 👋  Can I have your approval, please? 😄